travis/xenial/build.sh

```diff
@@ -26,3 +26,27 @@ if [ -n "${DEPLOY:+1}" ]; then
     cd /travis
     python3 setup.py sdist
 fi
+
+if [ -n "${BUILD_DIST:+1}" ]; then
+    mkdir -p /travis/results # /travis is persistent
+
+    cd /travis/travis
+    tar xvf keys.tar
+    gpg --import public.gpg || true
+    gpg --import private.gpg || true
+
+    # /build is inside the Docker container. we don't want to litter the host with persistent files. Especially useful if this is your dev machine
+    # /build is already made by the Dockerfile, so let's prepare it for use by a normal user
+    sudo chown -R $(id -u):$(id -g) /build
+    cd /build
+    pip3 download --no-deps pyqrllib
```

Comment:
same as before. This is getting an old pyqrllib. The publish stage has not run yet.